model-optimizer/mo/front/tf/extractors/strided_slice.py

```diff
@@ -31,6 +31,7 @@ def extract(cls, node):
             'ellipsis_mask': int_to_array_bit_mask(pb.attr["ellipsis_mask"].i),
             'new_axis_mask': int_to_array_bit_mask(pb.attr["new_axis_mask"].i),
             'shrink_axis_mask': int_to_array_bit_mask(pb.attr["shrink_axis_mask"].i),
+            'reinterp_shape': True,
```

Comment:
should be in a Op itself. Not in the extractor

Reply:
And TF has specific layout permutation of StridedSlice, so Transposes before and after SS won't be taken into the account while masks permutation